docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -27,7 +27,7 @@ from the top level `alluxio` directory with the following command:
 
 {% include Running-Spark-on-Alluxio/spark-profile-build.md %}
 
-* Add the following line to `spark/conf/spark-env.sh`.
+* Add the following line to `spark/conf/spark-defaults.conf`.
```

Comment:
I guess I was wondering if the conf file `spark-defaults.conf` works for both versions and I think you have answered my questions.